From 2283817b9aed0b6228d0255d5aba7968379c9854 Mon Sep 17 00:00:00 2001
From: Gilles Talis <gilles.talis@gmail.com>
Date: Wed, 3 Jul 2013 11:40:36 -0700
Subject: [PATCH] fbgrab: A proper Makefile for cross compiling
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Respect to the CC, CFLAGS and LDFLAGS is required for cross compiling in
Buildroot. And there's no need to run the source through splint.

Reworked original patch from: Daniel Nyström <daniel.nystrom@timeterminal.se>

Signed-off-by: Daniel Nyström <daniel.nystrom@timeterminal.se>
Signed-off-by: Gilles Talis <gilles.talis@gmail.com>
---
 Makefile |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index aae16e6..2d6b6fc 100644
--- a/Makefile
+++ b/Makefile
@@ -3,8 +3,10 @@
 ### modular. So this is a simple gnu Makefile...
 ###
 
-fbgrab: fbgrab.c
-	gcc -g -Wall fbgrab.c -lpng -lz -o fbgrab
+LDFLAGS += -lpng -lz
+
+fbgrab: fbgrab.o
+	$(CC) $(LDFLAGS) fbgrab.o -o $@
 
 install:
 	install fbgrab /usr/bin/fbgrab
-- 
1.7.4.1

